CMakeExt/HDF5.cmake

```diff
@@ -61,10 +53,23 @@ find_package_handle_standard_args(
   HDF5_INCLUDE_DIRS
 )
 
+
 # set flags
 if(HDF5_FOUND)
-#  set (HDF5_LINKER_FLAGS "-lhdf5_hl -lhdf5 -ldl -lm -lz")
-   set (HDF5_LINKER_FLAGS $ENV{HDF5_LIB} ${SZIP_LIB} -lz)
+  CHECK_SYMBOL_EXISTS(
+    H5_HAVE_PARALLEL
+    "${HDF5_INCLUDE_DIRS}/H5pubconf.h"
```

Comment:
I think we can solve this absolute path think with something like:

```
set(CMAKE_REQUIRED_INCLUDES ${HDF5_INCLUDE_DIRS})
```

At least @devreal is doing this in #396.